slides/week1/week1.org

```diff
@@ -130,6 +130,16 @@ Run some demos in your terminal! Ideally you should use each of these commands a
 | =mv= | Move a file |
 | =man= | Access documentation about a command |
 | =grep= | Search for a string |
+| =history= | Shows your command history |
```

Comment:
This is causing the slides to overflow on my computer. Can we bump the margins a bit or increase the resolution? Some of the intro slides also overflow for me.

Otherwise, we can split this into two slides.